stable/nginx-ingress/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: nginx-ingress
-version: 1.7.0
+version: 1.7.1
```

Comment:
We should bump the minor version here, so bump the version to 1.8.0.
Helm uses [semver](https://semver.org/) which is major.minor.patch

Reply:
Ah, sorry about that. Just updated :)